### PR TITLE
KEP for Cloud Provider DigitalOcean

### PR DESCRIPTION
**Pull request by andrewsykim**
KEP proposing DigitalOcean as a Kubernetes cloud provider.

### CODE REVIEW

File: keps/sig-cloud-provider/providers/0017-cloud-provider-digitalocean.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+DigitalOcean is a cloud computing platform built for developers and businesses who want a simple way to deploy & manage their infrastructure. This is a KEP proposing DigitalOcean as a cloud provider within the Kubernetes ecosystem.
+
+## Motivation
+
+### Goals
+
+* Supporting DigitalOcean within the Kubernetes ecosystem. This involves:
+  * providing an open community to promote discussions for Kubernetes on DigitalOcean
+  * providing an open environment to development Kuberentes on DigitalOcean
```
Comment (neolit123) [inline keps/sig-cloud-provider/providers/0017-cloud-provider-digitalocean.md]: `Kuberentes` -> `Kubernetes` :)

Comment (andrewsykim) [reply keps/sig-cloud-provider/providers/0017-cloud-provider-digitalocean.md]: ty :) surprised the spell checker didn't catch that :P

**Comment by hogepodge** [comment]
This all looks good, repository seems complete and ready to go.

/lgtm
/approve

**Comment by andrewsykim** [comment]
FYI this KEP is still in the `provisional` status, see https://github.com/kubernetes/community/pull/2449

**Pull request merged**